app/src/constants/routes.tsx

```diff
@@ -19,15 +19,16 @@
  */
 
 import {ProjectID, RecordID, RevisionID} from '@faims3/data-model';
+import {NOTEBOOK_NAME} from '../buildconfig';
 
 export const INDEX = '/';
 export const SIGN_IN = '/signin/';
 
 export const NOT_FOUND = '/not-found';
 export const WORKSPACE = '/workspace';
 
-export const NOTEBOOK = '/notebooks/';
-export const NOTEBOOK_LIST = '/notebooks';
+export const NOTEBOOK = `/${NOTEBOOK_NAME}s/`;
```

Comment:
@luke-mcfarlane-rocketlab  does this variable name make sense to you? I know you didn't choose it but feel free to update as you go - maybe "INDIVIDUAL_NOTEBOOK_ROUTE" and then "NOTEBOOK_LIST_ROUTE". We should be able to tell what the variable is without the context of it being imported from the routes file imo